features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/internal/TracingFeature.kt

```diff
@@ -33,6 +36,7 @@ internal class TracingFeature constructor(
 ) : StorageBackedFeature {
 
     internal var dataWriter: Writer = NoOpWriter()
+    internal var otelDataWriter: com.datadog.trace.common.writer.Writer = NoOpOtelWriter()
```

Comment:
alternatively you can use `as` in the import to avoid clashing and using full name.

Also now we have `com.datadog.legacy.trace.common.writer.Writer` and `com.datadog.trace.common.writer.Writer` - what is the difference? I see that in `develop` branch `TraceWriter` implements `com.datadog.trace.common.writer.Writer`.

Reply:
I moved the old classes last week (the ones in develop) in the legacy package in order to keep better track on what I have to remove later when we will only default on the `CoreTracer`

Reply:
thanks for the explanation! clear enough. I missed this part about moving the classes.